.github/scripts/run-benchmark.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+set -eo pipefail
+
+NUM_ITER=8
+NUM_ROUNDS=20
+DATA_JSON_PREFIX=$(date +"%Y%m%d_%H%M%S")
+BENCHMARK_FILTER="(not slomo)"
+
+# This machine has 16 physical cores, we use 12 of them to test
+CORE_LIST="4-15"
+export GOMP_CPU_AFFINITY="${CORE_LIST}"
+export CUDA_VISIBLE_DEVICES=0
+
+NO_TURBO_FILE="/sys/devices/system/cpu/intel_pstate/no_turbo"
+
+conda init bash; conda run /bin/bash
+
+if [[ -e ${NO_TURBO_FILE} ]]; then
+    sh -c "echo 1 > ${NO_TURBO_FILE}"
+fi
+
+pushd /workspace/benchmark
+
+for c in $(seq 1 $NUM_ITER); do
```

Comment:
just a couple of comments on the iterations/rounds.

- do you have a justification for running multiple (8) iterations? it will cost more, might not be needed?
- keep in mind that min-rounds is only minimum.  so if you set 20, some faster benchmarks will still run 100s of rounds.  This only matters later when uploading the data, need to be careful of not overwhelming the database. the uploader script has a limit currently.  just something to be aware of.

Reply:
Each iteration (20 rounds) costs about 50 mins to finish.
Therefore, 8 iterations is about 400 mins, or ~7 hours. This leaves us some time to bisect problematic PRs if we find some problem and we get enough data points to be confident that the benchmarking scores are stable.

Reply:
a related question is, what is the difference between doing 8 iterations of 20 rounds vs 1 iteration of 160 rounds?  

we should probably determine two things
 - how many rounds do we need for good signal?  (20? 40? 160?)
 - is there an important reason to break into multiple iterations? or just leave them together for simplicity?

Reply:
Good question. For now we run for 5 iterations and 20 rounds to make sure the score is stable. But more testing is required to determine the minimum round/iterations required to get a reasonable score.